devel/docker/packages

```diff
@@ -0,0 +1,28 @@
+FROM registry.fedoraproject.org/fedora:27
+
+RUN dnf install -y TurboGears2 python-moksha-wsgi intltool koji python2-fedora\
+    python-feedparser python-iniparse pytz python2-pyOpenSSL httpd mod_wsgi diffstat\
+    fedpkg python-lockfile python-tw2-jqplugins-ui python-bugzilla\
+    xapian-bindings-python python-dogpile-core python-dogpile-cache\
+    python-memcached python2-markdown pygobject3 fedmsg\
+    python2-pdc-client python2-webhelpers libappstream-glib dumb-init python2-GitPython git\
+    && dnf clean all
+
+RUN mkdir -p  /var/log/fedoracommunity\
+    /var/cache/fedoracommunity/packages/icons\
+    /var/cache/fedoracommunity/packages/xapian/search\
+    /var/cache/fedoracommunity/packages/tmp/icons\
+    /var/cache/fedoracommunity/packages/tmp/search\
+    /var/cache/fedoracommunity/packages/tmp/var\
+    /var/tmp/fedoracommunity/icons/tmp\
+    /usr/share/fedoracommunity
+
+#forward error logs to docker log collector
+RUN ln -sf /dev/stdout /var/log/httpd/fedoracommunity_error_log
+
+EXPOSE 80
+
+COPY development.ini /etc/fedoracommunity/production.ini
+COPY fedora-packages.conf /etc/httpd/conf.d/fedora-pacakges.conf
+ADD xapian.tar.gz /var/cache/fedoracommunity/packages
```

Comment:
I'd place this above since it may change less often than the config files

Also, can't both COPY line be merged?

Reply:
I have changed the order, but I can't seems to find a way to merge the COPY. It would work if the destination was the same which is not the case here.